content/departments/engineering/teams/source/strategy.md

```diff
@@ -1,3 +1,23 @@
-# Source Department Strategy
+# Source Strategy & Mission
 
-TL;DR: The Source department builds value for Sourcegraph by ensuring customers can index all of their code regardless of code host. We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
+## Strategy
+
+The Source team builds value for Sourcegraph by ensuring customers can index and secure all of their code regardless of code host and auth provider.
+
+We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
+
+## Mission
+
+The Source teams is responsible for the syncing, storing, and securing of code in Sourcegraph's code AI platform. We are putting the Source in Sourcegraph.
+
+We are building a foundation on which the rest of Sourcegraph stands. No other part of Sourcegraph can work without source code and properly managed access to it.
+
+Source makes Sourcegraph's code AI platform _universal_, by allowing it to access any code host, through any authentication and authorization provider, that customers require.
+
+We are building a product for site-admins and a platform for engineers at Sourcegraph.
+
+What we build is reliable, predictable, maintainable, debuggable.
+
+Sourcegraph site-admins trust the tools we build for them. They enjoy using them, because they help them get their job done faster and more easily, with less ambiguity.
```

Comment:
```suggestion
Sourcegraph site-admins trust and enjoy using the tools we build because they can easily manage them. Our goal is to provide a smooth, intuitive product that just works.
```

I like the below phrasing that engineers don't have to know/worry about the underlying source-related things work. I was thinking about putting a similar tone for the admin experience too.

Reply:
Incorporated 2nd part, I think the first part I'll leave as is because they shouldn't "manage the tools", they should use the tools to get a job done. I think we're in alignment though and it's just wording.